### ISSUE TITLE
Language other than English fails script

### ISSUE THREAD

**Issue by fleury08** [wimpysworld/rolling-rhino#19]
Running script

> $sudo ./rolling-rhino
> Rolling Rhino 🦏
>   [+] INFO: lsb_release detected.
>   [+] INFO: Ubuntu detected.
>   [+] INFO: Ubuntu Groovy Gorilla (development branch) detected.
>   [!] ERROR: No installed desktop packages were detected. Quitting.

Testing part where It crashed
> $apt list --installed ubuntu-desktop | grep installed  
> WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

I'm using Czech language and changing "installed" to "instalovaný" in script works
> apt list --installed ubuntu-desktop | grep instalovaný
> WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
> ubuntu-desktop/groovy,now 1.451 amd64 [instalovaný]

**Comment by m-ueberall** [comment]
Changing [line 92](https://github.com/wimpysworld/rolling-rhino/blob/8a12653f487014c5323af18351192d141b2b2a4a/rolling-rhino#L92) to
```
  INSTALLED=$(LANG=C apt list --installed "${META}" 2>/dev/null | grep installed)
```
should fix this for everyone.

**Comment by flexiondotorg** [comment]
Fixed via #15
